apps/tests/integration/integration_test.go

```diff
@@ -53,7 +55,9 @@ func httpRequest(ctx context.Context, url string, query url.Values, accessToken
 		return nil, fmt.Errorf("failed to build new http request: %w", err)
 	}
 	req.Header.Set("Authorization", "Bearer "+accessToken)
-	req.URL.RawQuery = query.Encode()
+	if querry := query.Encode(); querry != "" {
```

Comment:
`url.Values{}.Encode()` is okay, it just returns an empty string, which is the zero value of `url.RawQuery` anyway

Reply:
Updated.